packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -959,7 +959,7 @@ class _ScaffoldLayout extends MultiChildLayoutDelegate {
       final ScaffoldPrelayoutGeometry currentGeometry = ScaffoldPrelayoutGeometry(
         bottomSheetSize: bottomSheetSize,
         contentBottom: contentBottom,
-        contentTop: contentTop,
+        contentTop: appBarHeight,
```

Comment:
This looks like the correct change however it's a little confusing because the `contentTop` variable still exists and the ScaffoldPrelayoutGeometry parameter is called contentTop. It would help to add a comment here that briefly explains what's going on.

Reply:
@HansMuller 
I added comment for using `appBarHeight` instead of `contentTop`.
Thanks for your reply!